### PR TITLE
fix:  View.propTypes style issue + upgrade rn and dev dependencies

### PR DESCRIPTION
**Pull request by totorototo**
fix react-native-elements issue (android prod).

### CODE REVIEW

**Comment by guipasmoi** [comment]
we should remove that dependency

**Comment by totorototo** [comment]
@guipasmoi I guess you're right! 
we should get ride of it asap!

File: app/components/specific/countdown/Countdown.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Countdown extends Component {
       .isRequired,
     now: PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(Date)])
       .isRequired,
-    timerStyle: View.propTypes.style
+    timerStyle: PropTypes.style
```
Comment (guipasmoi) [inline app/components/specific/countdown/Countdown.js:16]: this shouldn't exist

Comment (guipasmoi) [reply app/components/specific/countdown/Countdown.js:16]: i am starting to think passing a style is a code smell

**Review by guipasmoi — APPROVED** [review]

**Pull request merged**